homeassistant/components/ipma/weather.py

```diff
@@ -56,14 +56,36 @@
 
 async def async_setup_platform(hass, config, async_add_entities,
                                discovery_info=None):
-    """Set up the ipma platform."""
+    """Set up the ipma platform.
+
+    Deprecated.
+    """
+    _LOGGER.warning('Loading IPMA via platform config is deprecated.')
```

Comment:
Please remove period at the end of the logging message.

Reply:
this was literally copy&paste from Sonos 😄